benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py

```diff
@@ -164,15 +173,15 @@ def run_once(model: Callable, inp: InputsType, task: str, v: VType, **kwargs) ->
     else:
         res = func(model, inp, strict=True)
 
-def run_once_functorch(model: Callable, inp: InputsType, task: str, v: VType, check_consistency=False) -> None:
+def run_once_functorch(model: Callable, inp: InputsType, task: str, v: VType, maybe_check_consistency=False) -> None:
```

Comment:
We should leave a quick comment that only run_once_functorch checks consistency otherwise other people will be confused

Reply:
How about https://github.com/pytorch/pytorch/pull/75689/files#diff-869a8c231e9cb0c1d3a62c44e108637892b034de902d0d814fad87c3d2105999R209-R210 ?